networking/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -26,4 +26,38 @@ The Network Observability Operator is now stable and the release channel is upgr
+
+[id="network-observability-operator-1.2.0-bug-fixes"]
+=== Bug fixes
+
+* Previously, after changing the `namespace` value in the FlowCollector spec, eBPF Agent pods running in the previous namespace were not appropriately deleted. Now, the pods running in the previous namespace are appropriately deleted. (link:https://issues.redhat.com/browse/NETOBSERV-774[NETOBSERV-774])
+
+* Previously, after changing the `caCert.name` value in the FlowCollector spec (such as in Loki section), FlowLogs-Pipeline pods and Console plugin pods were not restarted, therefore they were unaware of the configuration change. Now, the pods are restarted, so they get the configuration change. (link:https://issues.redhat.com/browse/NETOBSERV-772[NETOBSERV-772])
+
+* Previously, network flows between pods running on different nodes were sometimes not correctly identified as being duplicates because they are captured by different network interfaces. This resulted in over-estimated metrics displayed in the console plugin. Now, flows are correctly identified as duplicates, and the console plugin displays accurate metrics. (link:https://issues.redhat.com/browse/NETOBSERV-755[NETOBSERV-755])
+
+* Previously, the "reporter" option in the console plugin, which is used to filter flows based on the observation point of either source node or destination node, mixed the flows regardless of the node observation point. This was due to network flows being incorrectly reported as Ingress or Egress at the node level.  Now, the network flow direction reporting is corrected. The "reporter" option filters for source observation point, or destination observation point, as expected. (link:https://issues.redhat.com/browse/NETOBSERV-696[NETOBSERV-696])
+
+* Previously, for agents configured to send flows directly to the processor as GRPC+protobuf requests, the submitted payload could be too large and is rejected by the processors' GRPC server. This occured under  very-high-load scenarios and with only some configurations of the agent. The agent logged an error message, such as: _grpc: received message larger than max_. As a consequence, there was information loss about those flows.
+Now, the GRPC payload is split into several messages when the size exceeds a threshold. As a result, the connectivity is maintained. (link:https://issues.redhat.com/browse/NETOBSERV-617[NETOBSERV-617])
+
+* Previously, custom namespaces were supported. Custom namespaces are no longer supported. You cannot automatically upgrade to the new Operator version if you previously installed the Network Observability Operator using a custom namespace. You must delete the instances of the Operator using the cusom namespace and re-install your operator in the `openshift-netobserv-operator` namespace. (link:https://issues.redhat.com/browse/NETOBSERV-907[NETOBSERV-907])
```

Comment:
> Previously, custom namespaces were supported.

It misses the precision that this is for the operator installation - it's important because custom namespaces are still possible for the FlowCollector related deployments (ie what you set in `spec.namespace`).
So basically:
- Operator must be in `openshift-netobserv-operator` namespace.
- Others (FLP, Agent, plugin...) can be customized

Reply:
@jotak AH, got it. Addressed.

Reply:
@skrthomas typically we use the `netobserv` namespace for those pods (we also use this NS for Loki/Kafka pods in QE) if you want to suggest an example, but +1 to what @jotak said these resources can be anywhere so long as things are properly configured

Reply:
Gotcha. I amended this 907 note quite a bit, so hopefully there's more clarity around namespaces, and why we changed things.

Reply:
Per discussion in Scrum and Slack, this bug fix note was moved to Notable Technical Updates